.ci/travis/linux/docker-build-test.sh

```diff
@@ -81,6 +81,14 @@ pushd /root/QGIS > /dev/null
 /root/QGIS/tests/testdata/provider/testdata_pg.sh
 popd > /dev/null # /root/QGIS
 
+##########################################################
+# Set Postgres server location for that tests that require
+# setting up a custom server and not use a standard 
+# dockerised one
```

Comment:
this one is also dockerized, we're running in a docker container here, the other one just runs in a dedicated docker ;)

Reply:
well.. sure I'll fix the comment